app/views/partials/foyer/capture-montant-ressource.html

```diff
@@ -29,14 +29,14 @@ <h3 class="panel-title">
         </span>
       </div>
       <div class="col-sm-3">
-        <label for="{{ index + ressource.type.id }}-annuel">
+        <label for="{{ ressource.type.id }}_annuel_{{ index }}">
           Total depuis {{ debutAnneeGlissante }}
         </label>
         <span class="input-group">
           <input
             type="number"
             class="form-control text-right"
-            id="{{ index + ressource.type.id }}-annuel"
+            id="{{ ressource.type.id }}_annuel_{{ index }}"
```

Comment:
`index` is here to identify the individu, not the year. I would suggest: 
`{{ ressource.type.id }}-month_{{ $index + 1 }}-individu_{{ index }}` and `{{ ressource.type.id }}-year-individu_{{ index }}`.

(The need to have an id per individu may disappear with the resource declaration split).